testapps/testapp/src/main/res/raw/msal_config_msa.json

```diff
@@ -1,6 +1,7 @@
 {
   "client_id" : "9668f2bd-6103-4292-9024-84fa2d1b6fb2",
   "authorization_user_agent" : "DEFAULT",
+  "webauthn_capable": true,
```

Comment:
Do we also have a test for not capable scenario?

Reply:
Added a couple of tests: c6a6f63b6b42e176d8c0108e8cea1875a3bacb5a
But noting that I removed this change in the MSA config files (was for testing; don't mean to merge this in)